### ISSUE TITLE
Out of index error

### ISSUE THREAD

**Issue by XLechter** [angeladai/sgnn#2]
Hi, there is an out of index error when I run the training code.
Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
File "/sgnn/torch/loss.py", line 67, in compute_bce_sparse_dense
    tgtvalues = dense_tgts.view(-1)[flatlocs]
The log shows dense_tgts.view(-1) is of size 8192 while [flatlocs] is of size 32768
Could you give any suggetsions? I use the default settings during the training with pytorch 1.4 and python 3.6.

**Comment by XLechter** [comment]
I fix the TODO part in train.py and I can run it now.

**Comment by LiuXinchen1997** [comment]
> I fix the TODO part in train.py and I can run it now.

Hi, I meet the same problem as u.
Could u plz tell me how u solve this problem? Thank u :-)

**Comment by XLechter** [comment]
I remember I fix the TODO part in train.py line 64:
    if '64-64-128' in args.data_path:
        args.input_dim = (128, 64, 64)
    elif '96-96-160' in args.data_path:
        args.input_dim = (160, 96, 96)
    if '64-64-64' in args.data_path:
        args.input_dim = (64, 64, 64)
just fix the input_dim to (128, 64, 64) and run it again.
I am not sure about the input_dim since it has been a long time. If it does not work, just try the other two dims.

发自 Outlook<http://aka.ms/weboutlook>
________________________________
发件人: Liu Xinchen <notifications@github.com>
发送时间: 2021年1月20日 16:12
收件人: angeladai/sgnn <sgnn@noreply.github.com>
抄送: WXZhang <zhangwxx@hotmail.com>; State change <state_change@noreply.github.com>
主题: Re: [angeladai/sgnn] Out of index error (#2)


I fix the TODO part in train.py and I can run it now.

Hi, I meet the same problem as u.
Could u plz tell me how u solve this problem? Thank u :-)

―
